src/applications/pre-need/config/form.jsx

```diff
@@ -82,6 +83,9 @@ const formConfig = {
   submitUrl: `${environment.API_URL}/v0/preneeds/burial_forms`,
   trackingPrefix: 'preneed-',
   transformForSubmit: transform,
+  downtime: {
+    dependencies: [externalServices.global],
```

Comment:
form configs that do not have services listed in a downtime config will use the catchall "global" service.